.github/workflows/ci.yaml

```
@@ -0,0 +1,30 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10-alpha']
```

There's no `3.10-alpha`. But you can hardcode [`3.10-alpha4`](https://github.com/actions/python-versions/releases). Still, I recommend using the range syntax in this case:
```suggestion
        python-version:
        - 3.6
        - 3.7
        - 3.8
        - 3.9
        - 3.10-alpha - 3.10.0
```

---

It still doesn't like it. :P

---

Looks like you need to add `4` too

---

oh.. or `.0`